roles/famedly_sync/defaults/main.yml

```diff
@@ -0,0 +1,30 @@
+famedly_sync_path: /opt/famedly-sync
+
+famedly_sync_container_image_force_pull: false
```

Comment:
`famedly_sync_container_image_force_pull: "{{ famedly_sync_container_image_tag is defined }}"`